src/ConditionalLogic.php

```diff
@@ -42,29 +44,49 @@ public static function where(string $name, string $operator, mixed $value = null
             '!=empty',
         ];
 
-        if (in_array($operator, $allowedOperators) === false) {
+        if (!in_array($operator, $allowedOperators)) {
             throw new InvalidArgumentException("Invalid conditional logic operator [$operator].");
         }
 
-        return new self($name, $operator, $value);
+        return new self($field, $operator, $value);
+    }
+
+    public function and(string|array $field, string $operator, mixed $value = null): static
+    {
+        $this->rules[] = $this->createRule($field, $operator, $value);
+        return $this;
+    }
+
+    private function createRule(string|array $field, string $operator, mixed $value = null): array
```

Comment:
I wonder if we shouldn't use a new attribute instead since we can take advantage of named arguments.

```php
ConditionalLogic::where('enable_highlight', '==', 'on', 'other-group')

ConditionalLogic::where(
	group: 'other-group'
	name: 'enable_highlight', 
	operator: '==', 
	value: 'on', 
)
```

Another idea is to create a new method for group.

```php
ConditionalLogic::group('other-group')->where('enable_highlight', '==', 'on', 'other-group')

// OR

ConditionalLogic::whereGroup('other-group', 'enable_highlight', '==', 'on', 'other-group')
```

Not really sure which way to go.

Reply:
The problem with this approach is that you can't define a group for the field which follow in a and() method

Reply:
Hmm, I'm guessing you'll need to specify the group in each where and and statement?

Reply:
So this approch would be great :) 

```
ConditionalLogic::where(
	group: 'other-group',
	name: 'enable_highlight', 
	operator: '==', 
	value: 'on', 
)
```

Reply:
Done :) https://github.com/vinkla/extended-acf/pull/122/commits/f421774f4dc03d4289c8244a85ab2f5845299ec9